packages/node/src/local/client.ts

```diff
@@ -94,6 +94,32 @@ export class LocalEvaluationClient {
     );
   }
 
+  /**
+   * Locally evaluate varints for a user.
+   *
```

Comment:
should have description about how it differs from v1 evaluate? (like for fetchv2)